### ISSUE TITLE
Global Table Support

### ISSUE THREAD

**Issue by er1c** [silvermine/dynamodb-table-sync#4]
When syncing to/from dynamodb global tables, the diffing should ignore the additional global-table related fields.

I technically haven't tested this, simply looking at the code trying to figure out if this would work for global table replication, I think this would fail.  I think the diffing would need to ignore any of the `aws:` prefixed fields.  For reference: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/globaltables_HowItWorks.html

# Example

```json
{
  "aws:rep:deleting": {
    "BOOL": false
  },
  "aws:rep:updateregion": {
    "S": "us-west-2"
  },
  "aws:rep:updatetime": {
    "N": "1542916362.440001"
  },
  "created_on": {
    "N": "1539334691"
  },
  "id": {
    "S": "CatalogId-B2ZyEvviYV5CdgBwjbAN6"
  },
  "key": {
    "S": "catalog.autocare_webservice_enabled"
  },
  "updated_on": {
    "N": "1539334691"
  },
  "value": {
    "BOOL": false
  }
}
```

The work around would simply be to specifically use your `--ignore-att` flags, but the new "official" attributes should probably be added to the default list.  (I'm not sure if this is best handled in the `cli,js` or part of the `Synchronizer.js` in the `omit` function?

**Comment by jthomerson** [comment]
Haha, when I saw your issue I thought "wait, I already fixed that". I had a local test branch where I'd tested the changes, but I never pushed it. 🤦‍♂️I'll get that done soon and cut a new release.

**Comment by jthomerson** [comment]
@er1c could you possibly test the `repl-fields` branch to make sure it works for you? Unfortunately we don't have good automated testing on this project, and I'm pretty sure when I made this commit back in May I did the manual testing, but I won't have time this week to do any manual testing of it, and it sounds like you have a usecase readily available.

**Comment by er1c** [comment]
@jthomerson I'm using it to create new "environment-specific" tables (staging, test, personal dev, etc)

The idea would be to copy our production tables into our stage table.

# Working Example: 

```
dynamodb-table-sync \
--master us-west-2:attribute_values_development \
--slave us-west-2:attribute_values_eric \
--ignore-attr "aws:rep:deleting" \
--ignore-attr "aws:rep:updateregion" \
--ignore-attr "aws:rep:updatetime" \
--write-missing \
--write-differing \
--delete-extra
```

**Comment by jthomerson** [comment]
Yeah, that makes sense. Actually, in thinking back to why I originally authored the commit in question, it wasn't for comparison as I originally thought - it was exactly what you're doing - provisioning a new environment and seeding it with data from PRD.
